### PR TITLE
user can view all folders from api

### PR DESCRIPTION
**Pull request by stovermc**
#### What does  this PR do?
This allows an authenticated user to view retrieve all of their folders from the api
#### Where should the reviewer start?
In order to mock this functionality, an authentication token must be generated in the terminal. This process will complete the OAuth handshake in order to authenticate a user.
#### How should this be manually tested?
Once the access token has been retrieved from the terminal use the following url
localhost:3000/api/v1/sharing/folders?access_token= < access token from terminal >
#### Questions:
  - Do Migrations Need to be ran? 
  Yes -  Doorkeeper  Gem created many
  - Do Environment Variables need to be set? 
  
  - Any other deploy steps?

### CODE REVIEW

**Review by VictoriaVasys — APPROVED** [review]
Looks great, nice work! Need to update all landing_page_path instances to root_path

File: app/controllers/users/folders/binaries_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def new
   end
 
   def create
+    binding.pry
```
Comment (jmejia) [inline app/controllers/users/folders/binaries_controller.rb:10]: Whoops!

File: config/locales/doorkeeper.en.yml [hunk 2]
```diff
@@ -0,0 +1,124 @@
+en:
```
Comment (jmejia) [inline config/locales/doorkeeper.en.yml:1]: NICE! INTERNATIONALIZATION!

**Review by jmejia — COMMENTED** [review]
Great work @stovermc (Sorry for the delayed PR review!)

**Pull request merged**